SPECS/intel-gpu-device-plugin/namespace.patch

```diff
@@ -0,0 +1,11 @@
+# This patch sets the namespace to 'kube-system'.
```

Comment:
this just describes what the change does. could you add why as well so we don't forget later on?

Reply:
Done